source/assets/sass/config/_scale.scss

```diff
@@ -2,27 +2,58 @@
 // -------------
 /// @group scale
 
-$gutter: 1rem; // 16
-$gutter-quarter: calc(var(--gutter) * 0.25); // 4
-$gutter-half: calc(var(--gutter) * 0.5); // 8
-$gutter-minus: calc(var(--gutter) * 0.75); // 12
-$gutter-plus: calc(var(--gutter) * 1.25);
-$gutter-sesqui: calc(var(--gutter) * 1.5);
-$gutter-double: calc(var(--gutter) * 2);
-$gutter-triple: calc(var(--gutter) * 3);
+$gutter: 1rem; // ~16px
+$gutter-quarter: calc(var(--gutter) * 0.25); // ~4px
+$gutter-half: calc(var(--gutter) * 0.5); // ~8px
+$gutter-minus: calc(var(--gutter) * 0.75); // ~12px
+$gutter-plus: calc(var(--gutter) * 1.25); // ~20px
+$gutter-sesqui: calc(var(--gutter) * 1.5); // ~24px
+$gutter-double: calc(var(--gutter) * 2); // ~32px
+$gutter-double-sesqui: calc(var(--gutter) * 2.5); // ~40px
+$gutter-triple: calc(var(--gutter) * 3); // ~48px
 
 // Negative Spacing Sizes
 // ----------------------
+/// @group scale
```

Comment:
Is SassDoc used in this project?

Reply:
There's one example on the documentation / comments page but I added this mostly out of habit. I can remove it. I don't think it is hurting anything other than having more unused code.